### PR TITLE
Fade in the image-selector images when they change

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**

### CODE REVIEW

**Comment by CodeBaboon** [comment]
Code LGTM

**Comment by jstefaniuk-d2l** [comment]
Design approved by Bob

**Comment by jstefaniuk-d2l** [comment]
After this PR I'll be doing a second phase where I break out the image stuff into it's own polymer component and people can get the nice srcset and fade-in behaviour anywhere

**Comment by CodeBaboon** [comment]
No card in Rally to move, but I will test this.

**Comment by CodeBaboon** [comment]
This looks good on Mac in Chrome 54 and Safari 10. I can't get FireFox to load my dev instance properly at all (no errors in console, hard refresh doesn't fix) so I can't verify that browser.

File: image-selector/d2l-image-selector-tile.html [hunk 1]
```diff
@@ -9,7 +9,22 @@
...
+				opacity: 0;
+			}
+
+			.shown {
+				animation-name: shown;
+				animation-duration: 0.5s;
+				animation-fill-mode: forwards;
+			}
+
+			@keyframes shown {
```
Comment (dbatiste) [inline image-selector/d2l-image-selector-tile.html]: Ok cool.  I might be able to clean up some of these in a couple other components.

**Comment by CodeBaboon** [comment]
Tested using Mac FireFox 49 against Jeff's instance and it looks great.

**Pull request merged**